packages/config/config/devices/0x0371/zw175.json

```diff
@@ -6,295 +6,234 @@
-			"valueSize": 2,
-			"minValue": 0,
-			"maxValue": 1,
-			"defaultValue": 0
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "React to Alarm Type: Access Control Alarms",
+			"valueSize": 2
 		},
 		"9[0x4000]": {
-			"label": "Alarm reaction to Home Security Alarms",
-			"description": "React to Home Security Alarms from other Z-Wave devices.",
-			"valueSize": 2,
-			"minValue": 0,
-			"maxValue": 1,
-			"defaultValue": 0
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "React to Alarm Type: Home Security Alarms",
+			"valueSize": 2
 		},
-		"10": {
-			"label": "Release/Disable alarm",
-			"description": "Sets how you are able to disable Smart Switch 7s alarm state",
+		"8": {
+			"label": "Switch Action on Alarm",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 0,
-			"unsigned": true
-		},
-		"18": {
-			"label": "Flash frequency",
-			"description": "Configures the LED flash frequency and amount of time",
-			"valueSize": 2,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 2
-		},
-		"19": {
-			"label": "Start/Stop LED flash",
-			"description": "Sets the timeframe of flashing in seconds",
-			"valueSize": 2,
-			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 3,
 			"defaultValue": 0,
-			"unit": "seconds",
-			"writeOnly": true
-		},
-		"20": {
-			"label": "Power restored state",
-			"description": "Switches state after an power outage",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 2,
-			"defaultValue": 0
-		},
-		"80": {
-			"label": "Status Update",
-			"description": "Determine which reports are sent to your gateway",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 2,
-			"defaultValue": 2,
+			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "No report sent",
+					"label": "Disable",
 					"value": 0
 				},
 				{
-					"label": "Report Basic Report",
+					"label": "Turn on",
 					"value": 1
 				},
 				{
-					"label": "Report Binary Switch",
+					"label": "Turn off",
 					"value": 2
+				},
+				{
+					"label": "Cyclce on/off in 5 second intervals",
+					"value": 3
 				}
 			]
 		},
-		"81": {
-			"label": "LED indicator mode",
-			"description": "Allows you to configure how the LED indicator works",
-			"valueSize": 1,
+		"10": {
+			"label": "Method to Disable Alarm",
+			"description": "Allowable range: 10-255 - defines alarm duration",
+			"valueSize": 2,
+			"unit": "seconds",
 			"minValue": 0,
-			"maxValue": 2,
-			"defaultValue": 2,
-			"allowManualEntry": false,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
 			"options": [
 				{
-					"label": "LED Disabled",
+					"label": "Tap action button 3x",
 					"value": 0
 				},
 				{
-					"label": "Night Light Mode",
+					"label": "Idle state from corresponding alarm",
 					"value": 1
-				},
-				{
-					"label": "On/Off Mode",
-					"value": 2
 				}
 			]
 		},
```

Comment:
This parameter confuses me. What does it do? Alarm duration or something else?

Reply:
Both. Duration or specifies which combination of buttons/reports.